src/main/flight/pid.h

```diff
@@ -234,6 +234,8 @@ typedef struct pidProfile_s {
     uint8_t tpa_mode;                       // Controls which PID terms TPA effects
     uint8_t tpa_rate;                       // Percent reduction in P or D at full throttle
     uint16_t tpa_breakpoint;                // Breakpoint where TPA is activated
+    uint8_t tpa_rate_lower;                 // Percent reduction in P or D at zero throttle
+    uint16_t tpa_breakpoint_lower;          // Breakpoint where lower TPA is deactivated
```

Comment:
new parameters should be added to the end of parameter groups

Reply:
Thx, done.